src/surge-xt/gui/SurgeGUIEditor.cpp

```diff
@@ -1509,6 +1509,8 @@ void SurgeGUIEditor::openOrRecreateEditor()
 
     // reset the lfo rate slider pointer
     lfoRateSlider = nullptr;
+    filterCutoffSlider[0] = nullptr;
+    filterCutoffSlider[1] = nullptr;
```

Comment:
You need to null the resonance pointrts too here right?

Reply:
Done